pkg/transformers/namespace.go

```diff
@@ -97,5 +97,46 @@ func (o *namespaceTransformer) Transform(m resmap.ResMap) error {
 		}
 
 	}
+	o.updateClusterRoleBinding(m)
 	return nil
 }
+
+func (o *namespaceTransformer) updateClusterRoleBinding(m resmap.ResMap) {
+	samap := resmap.ResMap{}
+	for id := range m {
+		if id.Gvk().Kind == "ServiceAccount" {
+			samap[id] = m[id]
+		}
+	}
+
+	for id := range m {
+		if id.Gvk().Kind == "ClusterRoleBinding" || id.Gvk().Kind == "RoleBinding" {
```

Comment:
Code readability comment: too many indents caused by deeply nested if statements.
you are doing:
```
for ... {
  if foo {
    do bar
  }
}
```
instead you can do
```
for ... {
  if !foo {
    continue
  }
  do bar
}
```
I guess this can make it easier to read.
Hope it makes sense.

Reply:
It makes sense. Updated.